scripts/publish.sh

```diff
@@ -424,6 +455,25 @@ create_manifest_list_init() {
 	docker manifest push ${1}:${tag}
 }
 
+create_manifest_list_init_debug() {
+
+	export DOCKER_CLI_EXPERIMENTAL=enabled
+	tag=${2}
+	version=${3}
+
+	# TODO: Add a way to automatically generate arch images in manifest
+	docker manifest create ${1}:${tag} ${1}:"$init"-arm64-debug-${version} ${1}:"$init"-amd64-debug-${version}
+
+	for arch in "${ARCHITECTURES[@]}"
+	do
+		docker manifest annotate --arch "$arch" ${1}:${tag} ${1}:"$init"-"$arch"-debug-${version}
+	done
+
+	# sanity check on the debug log.
+    docker manifest inspect ${1}:${tag}
```

Comment:
strange formatting. Would you please fix?

Reply:
this format is the same with create_manifest_list_init() function. I think this is the problem of github

Reply:
It's because there are spaces rather than a tab.

Reply:
Try delete the white space and use a tab instead like the rest of the whitespaces